appveyor.yml

```diff
@@ -6,11 +6,6 @@ environment:
       WITH_COMPILER: "cmd /E:ON /V:ON /C .\\extra\\appveyor\\run_with_compiler.cmd"
 
   matrix:
-    - TOXENV: "2.7"
```

Comment:
we will support py2.7 on 4.x series till the end of 2020 :)

Reply:
Well we don't today have a working appveyor build ever for it, and I have no way to know how to resolve that header error.  We should remove the appveyor build so they can pass in the PRs and add it back when it is working for py2.7 unless you know how to fix it.

Reply:
windows support is needed. there is a open pr toward kombu 5. we can remove it there :)

Reply:
This change doesn't remove py2 support, it just disables the broken appveyor build for it.  The configs still exist for Tox and Travis CI -- but it would let the appveyor output actually mean something, whereas right now those two configurations always fail at the same step which fails the whole build.

Reply:
@auvipy It would make sense to merge this and open a separate PR adding back the 2.7 appveyor build so only that PR would have the failure, and if it gets resolved we can merge back in support for a python 2.7 AppVeyor build.

Reply:
I will take more time to look into it. thanks for the effort.